.ci/Makefile

```diff
@@ -1,13 +1,16 @@
 SHELL = /bin/bash -eo pipefail
 
 AWS_FOLDER = ../build/aws
-LAMBDA_LAYER_ZIP_FILE = ../build/lambda-layer-zip/elastic-apm-node-lambda-layer.zip
+DIST_DIR = /build/dist/nodejs
```

Comment:
In Dockerfile it's always relative to the Dockerfile.
I first had the Dockerfile in the .ci sub dir and tried with relative `../build/dist/nodejs` but that does not work because all things added in the Dockerfile need to be on the same or below level in the directory structure relative to the Dockerfile